src/fzx/tui/term_app.cpp

```diff
@@ -63,11 +64,24 @@ void TermApp::processTTY()
       continue;
     }
     if (*key == kEnter || *key == kCtrlU) {
+      if (*key == kEnter) {
+        quit();
+        return;
+      }
       mLine.clear();
       updateQuery = true;
     } else if (*key == kCtrlC) {
       quit();
       return;
+    } else if (*key == kCtrlP) {
+      mCurpos++;
+    } else if (*key == kCtrlN) {
+      if (mCurpos > 0) {
+        mCurpos--;
+      }
+    } else if (*key == kTab) {
+      mSelection.push_back(mFzx.getResult(mCurpos).mLine);
```

Comment:
This string_view can get invalidated as new items come in, so if this is meant to always point at the same item get `.mIndex` (`uint32_t`), and then `mFzx.getItem` to get string_view back
```suggestion
      mSelection.push_back(mFzx.getResult(mCurpos).mIndex);
```
`mSelection` has to be then changed to `std::vector<uint32_t>`, and the `std::find` in `redraw()` should compare with `.mIndex` as well